common/djangoapps/student/models.py

```diff
@@ -2527,6 +2586,10 @@ class SocialLink(models.Model):  # pylint: disable=model-missing-unicode
     component of the stored URL and an example of a valid URL.
 
     The stored social_link value must adhere to the form 'https://www.[url_stub][username]'.
+
+    .. pii: Stores linkage from User to social media platforms. Retired in AccountRetirementView.
```

Comment:
I'd even specify here that it stores the link *to the user profile* on the social media platform.  That's highly specific.